bodhi/tests/server/base.py

```diff
@@ -19,70 +19,198 @@
 import os
 import unittest
 
+from webtest import TestApp
 from sqlalchemy import event
-import requests
+import mock
 
-from bodhi.server import bugs, buildsys, log, models, initialize_db, Session, config
+from bodhi.server import bugs, buildsys, models, initialize_db, Session, config, main
 from bodhi.tests.server import create_update, populate
 
 
 original_config = config.config.copy()
+engine = None
+_app = None
 
-DB_PATH = 'sqlite://'
-DB_NAME = None
 
+def _configure_test_db(db_uri='sqlite:////var/tmp/bodhi-tests.sqlite'):
```

Comment:
We could definitely make it relative to the source and that would be more portable. I wonder if pytest has some hook at the end of the tests to delete it... Nothing except the schema is written to the database, but it'd still be nice to be neat.